diff --git a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h
index c9db600..c631f83 100644
--- a/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h
+++ b/node_modules/@react-native-firebase/firestore/ios/RNFBFirestore/RNFBFirestoreCollectionModule.h
@@ -22,6 +22,8 @@
 #import <React/RCTBridgeModule.h>
 #import "RNFBFirestoreCommon.h"
 #import "RNFBFirestoreSerialize.h"
+#import <React/RCTBridgeModule.h>
+#import <React/RCTBridgeModule.h>
 
 @interface RNFBFirestoreCollectionModule : NSObject <RCTBridgeModule>
 
